app/brave_command_ids.h

```diff
@@ -14,6 +14,7 @@
 #define IDC_NEW_TOR_IDENTITY     56002
 #define IDC_NEW_OFFTHERECORD_WINDOW_TOR 56003
 #define IDC_CONTENT_CONTEXT_OPENLINKTOR 56004
+#define IDC_SHOW_BRAVE_SYNC      56007
```

Comment:
not a big deal, but did we skip numbers on purpose?

Reply:
probably caused by rebase.